lib/reports/BackupReportManager.js

```diff
@@ -134,7 +134,7 @@ class BackupReportManager {
         'data.instance_name': 1,
         'data.plan_id': 1,
         'data.service_plan_name': 1,
-        'data.space_guid': 1,
+        'data.tenant_id': 1,
```

Comment:
Can you please change the following to `tenant_id`
https://github.infra.hana.ondemand.com/cloudfoundry/service-fabrik-product-component/blob/master/scripts/reports/generate-backup-report#L118
and
https://github.infra.hana.ondemand.com/cloudfoundry/service-fabrik-product-component/blob/master/scripts/reports/generate-backup-report#L101

Reply:
done